src/components/common/Form.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import Autocomplete from 'react-autocomplete';
+
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedAddress: '',
+      isCollectionScheduleTextHidden: true,
+      isAutoTextHidden: false,
+      dateFormat: 'D/M/YYYY',
+      scheduleType: {
```

Comment:
in scheduleType if the property name is the same as the property value, you probably don't need this object. Instead you can just compare scheduleType in your logic.

Reply:
removed